version/1/4/ja/code_of_conduct.md

```diff
@@ -0,0 +1,68 @@
+# コントリビュータ協定 行動規範
+
+## 宣誓
+
+オープンで歓迎的な雰囲気の環境を育成するため、私達はコントリビュータお
+よびメンテナとして、年齢、体系、障碍、民族、性自認や性別表現、経験の多
+寡、国籍、容姿、人種、宗教、性的指向に関わりなく、私達のプロジェクトお
+よびコミュニティへの参加を誰にとっても嫌がらせのない体験にすることを誓
+います。
+
+## 基準
+
+ポジティブな環境を作り出すことに貢献する行動の例は以下のものがあります:
+
+* 歓迎的で開放的な言葉を使うこと
+* 異なる視点や経験を尊重する態度をとること
+* 建設的な批判を率直に受け入れること
+* 何がコミュニティにとって最適であるかに焦点を当てること
+* 他のコミュニティメンバーに共感すること
+
+受け入れられない行動の例としては以下のものがあります:
+
+* 性的な言葉・画像の使用および
```

Comment:
Oh yes I missed a part of the list item...I'll fix it soon. I should've checked more carefully...